<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QResultExprWidget_Range</class>
 <widget class="QWidget" name="QResultExprWidget_Range">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_btnClose">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(200, 0, 0);</string>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>结果参数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QResultExprWidgetButtonLine" name="m_editResultParas">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnResultParasDel">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QResultExprWidgetExprComboBox" name="m_cbbMinExpr">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="m_btnMinValueDel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="m_btnMaxValueDel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QResultExprWidgetButtonLine" name="m_editMaxValue">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QResultExprWidgetExprComboBox" name="m_cbbMaxExpr">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>基准-最大值</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QResultExprWidgetButtonLine" name="m_editMinValue">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>基准-最小值</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QResultExprWidgetExprComboBox</class>
   <extends>QComboBox</extends>
   <header>ResultExprWidgetItem.h</header>
  </customwidget>
  <customwidget>
   <class>QResultExprWidgetButtonLine</class>
   <extends>QLineEdit</extends>
   <header>ResultExprWidgetItem.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
